### PR TITLE
Adding cnmaps

### PR DESCRIPTION
**Pull request by Clarmy**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by carterbox** [comment]
@conda-forge/help-python please review

File: recipes/cnmaps/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{% set name = "cnmaps" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Clarmy/cnmaps/archive/refs/tags/0.2.1.tar.gz
```
Comment (carterbox) [inline recipes/cnmaps/meta.yaml:9]:
```suggestion
  url: https://github.com/Clarmy/cnmaps/archive/refs/tags/{{ version }}.tar.gz
```

File: recipes/cnmaps/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/Clarmy/cnmaps
+  summary: 'cnmaps is a python package that make drawing China maps more easily. '
+  description: |
+    cnmaps is a python package that make drawing China maps more easily. 
+  license: MIT
+  license_family: MIT
```
Comment (carterbox) [inline recipes/cnmaps/meta.yaml:43]: Remember to update the license label here when you update to a new version.

**Review by carterbox — APPROVED** [review]

File: recipes/cnmaps/meta.yaml (hunk 2 above)
Comment (Clarmy) [reply recipes/cnmaps/meta.yaml:43]: OK, Thank you.

**Pull request merged**